2000-10-18  Jan Hubicka  <jh@suse.cz>

	* i386.h (CONST_OK_FOR_LETTER): Add 'N'.

--- gcc/config/i386/i386.h.jj	Mon Oct 23 22:51:33 2000
+++ gcc/config/i386/i386.h	Fri Oct 27 17:26:40 2000
@@ -1052,6 +1052,7 @@ enum reg_class
    : (C) == 'K' ? (VALUE) >= -128 && (VALUE) <= 127		\
    : (C) == 'L' ? (VALUE) == 0xff || (VALUE) == 0xffff		\
    : (C) == 'M' ? (VALUE) >= 0 && (VALUE) <= 3			\
+   : (C) == 'N' ? (VALUE) >= 0 && (VALUE) <= 255		\
    : 0)
 
 /* Similar, but for floating constants, and defining letters G and H.
